index.html

```diff
@@ -257,7 +257,6 @@ <h1 id="contact-para">I'm always
 <input type="email"class="email" id="mail" placeholder="Email address" name="email" required>
         </label>
         <textarea name="textarea" id="text-area" cols="30" rows="10" maxlength="500" name="textarea" placeholder="Enter text here" required></textarea>
-        <br><span class="error"></span>
         <button id="Get-in-touch" type="submit">Get in touch
```

Comment:
- [ ] Kindly make sure your text area is visible on the page.

Your design :point_down: 

![image](https://user-images.githubusercontent.com/97100091/203916182-b1772fcf-033e-40b9-99b3-d6380dc4fcc6.png)

Figma template :point_down: 
![image](https://user-images.githubusercontent.com/97100091/203916449-5a83e07a-764c-4190-b0fc-5b800fac2c01.png)

Reply:
fixed issue